### PR TITLE
refactor(openid): remove openid login support

### LINKED ISSUE #1336: Remove openid login support
Like we're doing in content-server over in https://github.com/mozilla/fxa-content-server/pull/3920

### PR DESCRIPTION
**Pull request by vladikoff**
Fixes #1336

### CODE REVIEW

**Comment by vladikoff** [comment]
@rfk @dannycoates r?

**Comment by kitcambridge** [comment]
Pull request 1337! 😎

**Comment by seanmonstar** [comment]
\o/

File: lib/db.js [hunk 1]
```diff
@@ -432,24 +432,6 @@ module.exports = function (
       )
   }
 
-  DB.prototype.openIdRecord = function (id) {
-    log.trace({ op: 'DB.openIdRecord', id: id })
-    return this.pool.get('/openIdRecord/' + Buffer(id, 'utf8').toString('hex'))
```
Comment (rfk) [inline lib/db.js]: Does this mean we also have code we can kill in the db-server repo?  Sweet...

**Pull request merged**